lldb/include/lldb/Core/Progress.h

```diff
@@ -134,6 +144,9 @@ class Progress {
   /// Data needed by the debugger to broadcast a progress event.
   const ProgressData m_progress_data;
 
+  /// The origin of this progress event.
+  const Progress::Origin m_origin;
+
```

Comment:
This shouldn't be necessary given that the value is already accessible as `m_progress_data.origin`